spec/deck_spec.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/card'
+require './lib/deck'
+
+card_1 = Card.new("What is my name?", "Rontron", :People)
```

Comment:
These card variables should live below the 'Rspec.describe.' It's very useful to use 'let' blocks at the top of an Rspec file to set a variable for all tests: 
let!(:card1) { Card.new("What is my name?", "Rontron", :People) } 

https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let